### ISSUE TITLE
Play with materialize

### ISSUE THREAD

**Issue by Jeffail** [redpanda-data/connect#966]
It would be neat to experiment with different ways of integrating with https://github.com/MaterializeInc/materialize, inputs, outputs, processors, etc. There's a few different combinations here so ideally we want to be able to describe the generally neatest approach.

**Comment by gedw99** [comment]
I have been doing the same. I agree that it's a compelling combination.

Also it can work with cockroachdb perhaps, and not just Postgresql, but i haven't tested it.

**Comment by gedw99** [comment]
cc @danthegoodman1

**Comment by danthegoodman1** [comment]
Yeah I love what materialize is doing. One thing I wish to improve would be their outputs, I think combining benthos to empower their output selection would be ideal. Right now a simple redpanda (Kafka compatible) stream to stick outputs in benthos is the most efficient way I've tested.

**Comment by danthegoodman1** [comment]
I wonder if there is even a case to support basic operations such as AVGs, SUMs, etc. as a processor?

**Comment by danthegoodman1** [comment]
Also I was thinking for integration you could just use the `sql_insert` or `sql` outputs, since materialize supports in memory tables as well

This would obviously grow memory since there is not built-in TTL, but maybe a cron can be run to compact it (maybe even a benthos input could be cron...)

**Comment by twmb** [comment]
Closing as part of a backlog cleanup. Materialize is compatible with the Postgres wire protocol, so existing SQL components can be used for integration. If a dedicated integration is still needed, feel free to open a new issue or PR.
